src/meta_merge/core.cljc

```diff
@@ -1,11 +1,16 @@
 (ns meta-merge.core
   (:require [clojure.set :as set]))
 
+(defn metadatable?
```

Comment:
`metadatable?` uses the IMeta protocol to check for metadata now, and can be extended to check for the IWithMeta protocol.